packages/graphback-core/src/plugin/GraphbackPluginEngine.ts

```diff
@@ -57,15 +69,18 @@ export class GraphbackPluginEngine {
   }
 
   public createSchema(): GraphbackCoreMetadata {
-    if (this.plugins.length === 0) {
-      console.warn("GraphbackEngine: No Graphback plugins registered");
-    }
     //We need to apply all required changes to the schema we need
     //This is to ensure that every plugin can add changes to the schema
     for (const plugin of this.plugins) {
       const newSchema = plugin.transformSchema(this.metadata);
       this.metadata.setSchema(newSchema);
+    }
 
+    return this.metadata;
```

Comment:
`createSchema` and `createResolvers` could just be void functions, since there is no reason to return `this.metadata`, which is available as a class variable.

Reply:
Good point.

Reply:
Good point. Pushed a quick update.